roles/wazuh/ansible-wazuh-agent/tasks/installation_from_custom_packages.yml

```diff
@@ -39,21 +39,27 @@
     when: ansible_os_family == "Darwin"
 
 # Solaris
-  - name: Create noask file
-    lineinfile:
-      path: "/export/home/vagrant/noaskfile"
-      line: action=nocheck
-      create: yes
-    when: ansible_os_family == "Solaris"
+  - name: Install Solaris 10 wazuh agent custom package
+    become: true
+    shell: pkgadd -d wazuh*
+    environment:
+      PATH: "/opt/python3/bin/:/usr/sbin:/usr/bin:/usr/sbin/:/opt/csw/gnu/:/usr/sfw/bin/:/opt/csw/bin/"
+    args:
+      chdir: "/export/home/vagrant"
+    when:
+      - ansible_os_family == 'Solaris'
+      - wazuh_custom_packages_installation_agent_solaris_10_url is defined
 
-  - name: Install solaris wazuh agent custom package
+  - name: Install Solaris 11 wazuh agent custom package
     become: true
-    shell: pkgadd -a noaskfile -d wazuh-* -n all
+    shell: pkg install -g wazuh* wazuh-agent
```

Comment:
Why do we remove `noaskfile`? It is only needed for Solaris 10? Check that in the Solaris 10 package installation is also missing

Reply:
The `noaskfile` installation administration file is not required for any Solaris version. This installation logic was replicated from IT pipeline. However, it is better to follow the exacts [documentation page steps](https://documentation.wazuh.com/current/installation-guide/wazuh-agent/wazuh-agent-package-solaris.html#installing-wazuh-agents-on-solaris-endpoints). In this case, without the `noaskfile`.